pkg/runctl/controller.go

```diff
@@ -513,35 +516,40 @@ func (c *Controller) handlePipelineRunNew(
 	return false, nil
 }
 
-func (c *Controller) ensurePipelineRunsConfig(ctx context.Context, pipelineRun k8s.PipelineRun) (*cfg.PipelineRunsConfigStruct, bool, error) {
+var emptyLoggingDetais = map[string]cfg.PipelineRunAccessor{}
```

Comment:
Why not using `nil`?

Reply:
Changed: https://github.com/SAP/stewardci-core/pull/412/commits/6b7c08f4efca6fab957ec3c1ea8cc98c6b9ed1f8